src/MCPClient/tests/test_create_aip_mets.py

```diff
@@ -883,38 +883,55 @@ def test_create_file_sec(self):
             self.state.fileNameToFileID
         ), "State hasn't been generated for all objects on disk, duplicate names may not be counted for"
 
-    def test_get_included_structmap(self):
-        """Test the output of custom structmaps in create_mets_v2."""
+    def test_get_included_structmap_invalid_mets(self):
+        """Integration test ensuring that it is possible for the mets
+        validation to fail using the mets_xsd_path included with Archivematica.
+        """
+        self.generate_aip_mets_v2_state()
+        broken_structmap_path = os.path.join(
+            self.objects_dir,
+            "metadata",
+            "transfers",
+            "custom-structmap-41ab1f1a-34d0-4a83-a2a3-0ad1b1ee1c51",
+            "broken_structmap.xml",
+        )
+        assert os.path.isfile(broken_structmap_path)
+        assert os.path.isfile(self.mets_xsd_path)
+        try:
+            self.validate_mets(self.mets_xsd_path, broken_structmap_path)
+        except etree.DocumentInvalid:
+            assert (
+                True
```

Comment:
Nothing to do about this, but I'm never sure how to best represent these cases: `assert True`. A comment may have the same effect.

Reply:
That's a fair point. I might try a comment style the next time around; though I hope we move away more from test case and use the pytest style you showed me with the checksum validation microservice.